corehq/apps/sso/tests/test_backends.py

```diff
@@ -24,12 +24,15 @@ class TestSsoBackend(TestCase):
     def setUpClass(cls):
         super().setUpClass()
         cls.account = generator.get_billing_account_for_idp()
+        cls.addClassCleanup(cls.account.delete)
```

Comment:
Thank you! I'm working on this file and also plan to do the same thing. Now I can just wait for this PR get merged and do a rebase. 😂

Reply:
This has to go through QA, so might be a while before it gets merged. You could cherry-pick the commit that contains that into your PR if you want it sooner.